proposals/lip-0065.md

```diff
@@ -111,6 +111,7 @@ To summarize, we chose sparse Merkle trees because:
 | `EVENT_NAME_DEFAULT` | string | `commandExecutionResult` | Name of default event emitted at the end of transaction execution. |
 | `EVENT_INDEX_LENGTH_BITS` | integer | TBD | Length in bits of the index used to calculate events tree leaf keys. |
```

Comment:
Btw, don't we actually have a value for this constant?

Reply:
good point, I added all tbd constant values